services/dumper.js

```diff
@@ -64,7 +64,6 @@ function Dumper(config) {
       name: config.appName.replace(/ /g, '_').toLowerCase(),
       version: '0.0.1',
       private: true,
-      scripts: { start: 'node ./bin/www' },
```

Comment:
[Spec] We cannot remove this as it will break the start with nodemon

Reply:
I used to run `nodemon bin/www` and now I have to run `nodemon server.js`. Was there another (better) way?

Reply:
`nodemon` simply. `Nodemon` use the start by default ^^

Reply:
OK `nodemon` worked too.